common/transport/amqp/devdoc/amqp_message.md

```diff
@@ -77,20 +75,20 @@ The `toMessage` static method takes an AMQP message from the underlying library
 
 **SRS_NODE_IOTHUB_AMQPMSG_16_002: [** The `toMessage` method shall return a `Message` object. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_003: [** The `toMessage` method shall set the `Message.correlationId` property to the `AmqpMessage.properties.correlationId` value if it is present. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_003: [** The `toMessage` method shall set the `Message.correlationId` property to the `AmqpMessage.correlation_id` value if it is present. **]**
 
```

Comment:
no buffer translation on this part?

Reply:
Hmmm, perhaps there should be another requirement or two.  If the amqp property correlation_id is of type buffer AND is of length 16 it will do a rhea uuid type to string coversion.  Otherwise it will simply set the agnostic correlationId to the amqp correlation_id.

---
In reply to: [209124547](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209124547) [](ancestors = 209124547)